modules/vsphere-enabling-multiple-layer2-networks.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="vsphere-enabling-multiple-layer2-network_{context}"]
+= Enabling a multiple layer 2 network for your cluster
+
+You can configure your cluster to use a multiple layer 2 network configuration, so that data transfer among nodes can span across multiple networks. 
```

Comment:
```suggestion
You can configure your cluster to use a multiple layer 2 network configuration so that data transfer among nodes can span across multiple networks. 
```